custom/icds_reports/reports/adolescent_girls.py

```diff
@@ -204,7 +192,7 @@ def get_adolescent_girls_data_chart(domain, config, loc_level, show_test=False):
             }
         ],
         "all_locations": top_locations,
-        "top_three": top_locations[0:5],
-        "bottom_three": top_locations[-6:],
+        "top_three": top_locations[:5],
```

Comment:
I think this would be top 5 based on the code

Reply:
I will change this in all reports